mordant/build.gradle.kts

```diff
@@ -3,6 +3,7 @@
 plugins {
     kotlin("multiplatform")
     alias(libs.plugins.publish)
+    `java-library`
```

Comment:
```suggestion
```

No need for the java-library plugin, the kotlin plugin handles bundling the resources already.